src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,117 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
+  const [isLoading, setIsloading] = useState(false);
+  const [isErorrOnLoadPeople, setIsErorrOnLoadPeople] = useState(false);
+  const [isNoPeopleOnServer, setIsNoPeopleOnServer] = useState(false);
```

Comment:
Don't add extra variables to state. Use people.length === 0 instead isNoPeopleOnServer